src/components/reader/pager/VerticalPager.tsx

```diff
@@ -111,7 +112,7 @@ export function VerticalPager(props: IReaderProps) {
     }, [selfRef]);
 
     const go = useCallback(
-        (direction: 'up' | 'down') => {
+        (direction: 'up' | 'down', slightly = false) => {
```

Comment:
```suggestion
        (direction: 'up' | 'down', offset: number = SCROLL_OFFSET) => {
```
with this the comments for the constants can also be removed (`// left and right arrows` and `// up and down arrows`)